ably/export_test.go

```diff
@@ -106,3 +106,21 @@ func (c *Connection) OnState(ch chan<- State, states ...StateEnum) {
 func (c *Connection) OffState(ch chan<- State, states ...StateEnum) {
 	c.offState(ch, states...)
 }
+
+func (c *Connection) RecoveryKey() string {
+	c.state.Lock()
+	defer c.state.Unlock()
+	return c.recoveryKey
+}
+
+func (c *Connection) RemoveKey() {
+	c.state.Lock()
+	defer c.state.Unlock()
+	c.key = ""
+}
+
+func (c *Connection) MsgSerial() int64 {
```

Comment:
This isn't part of the public API defined in the spec. If you want to expose it for tests, you can put it in `export_test.go`.

Reply:
it is in `export_test.go` file already